packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -75,31 +72,12 @@ export class DeltaStorageService implements IDeltaStorageService {
         id: string,
         from: number,
         to: number): Promise<IDeltasFetchResult> {
-        const query = querystring.stringify({ from, to });
-
-        const headers: OutgoingHttpHeaders = {
-            "x-correlation-id": uuid(),
-        };
-
-        const storageToken = await this.tokenProvider.fetchStorageToken(
-            tenantId,
-            id,
-        );
-
-        if (storageToken) {
-            headers.Authorization = `Basic ${storageToken.jwt}`;
-        }
-
-        const ops = await Axios.get<ISequencedDocumentMessage[]>(
-            `${this.url}?${query}`, { headers });
-
-        this.logger.sendTelemetryEvent({
-            eventName: "R11sDriverToServer",
-            correlationId: headers["x-correlation-id"] as string,
-        });
+        const ordererRestWrapper = new RouterliciousOrdererRestWrapper(tenantId, id, this.tokenProvider, this.logger);
+        await ordererRestWrapper.load();
```

Comment:
What happens if this call fails?

Reply:
In my latest commit, I wrapped it in a try..catch that logs the error. Now, if it fails, it will still make the request, fail with a 401 (assuming correct service implementation), retry `getAuthHeader`, and either fail again there which will bubble up as a critical error or succeed and carry on.